build.gradle

```diff
@@ -97,9 +97,10 @@ subprojects {
   targetCompatibility = 1.6
 
   repositories {
-    if (project.hasProperty('mvnlocal')) {
-      mavenLocal()
-    }
+//    if (project.hasProperty('mvnlocal')) {
+//      mavenLocal()
+//    }
+    mavenLocal()
```

Comment:
You should drop that change and put `mvnlocal=true` in your `~/.gradle/gradle.properties` file before that gets merged by accident once again.